_layouts/default.html

```diff
@@ -9,7 +9,12 @@
   <link href="{{ "/assets/webclip.png" | relative_url }}" rel="apple-touch-icon">
 
   <link id="themeCSS" rel="stylesheet" href="{{ "assets/css/style.css" | relative_url }}">
-  <script defer src="https://pro.fontawesome.com/releases/v5.1.0/js/all.js" integrity="sha384-E5SpgaZcbSJx0Iabb3Jr2AfTRiFnrdOw1mhO19DzzrT9L+wCpDyHUG2q07aQdO6E" crossorigin="anonymous"></script>
+
+  {% if site.use_fa_pro %}
+  <script defer src="https://pro.fontawesome.com/releases/v5.8.1/js/all.js" integrity="sha384-E5SpgaZcbSJx0Iabb3Jr2AfTRiFnrdOw1mhO19DzzrT9L+wCpDyHUG2q07aQdO6E" crossorigin="anonymous"></script>
```

Comment:
You're right, I forgot to update the integrity variable, however, I do not have access to the Pro version so that can be reverted or the integrity updated.